clients/admin-ui/src/features/config-wizard/setup.tsx

```diff
@@ -19,15 +18,21 @@ const Setup = ({ wizardStep }: Props) => {
       >
         <Image
           boxSize="100%"
-          // fallback={}
-          objectFit="fill"
-          // src=""
+          objectFit="cover"
+          src="/config_splash.svg"
```

Comment:
I would suggest the relative path for this image to be `/images/config_splash.svg`. You can move this .svg to the public/images folder.